.gitignore

```diff
@@ -19,3 +19,6 @@ files/*
 .sass-cache
 static/css/*
 !static/css/.keep
+
+# Exclude flask config file as it contains secret keys
+instance/config.py
```

Comment:
Is this relevant to the docker image?

Reply:
nope. not really. but I think its useful to have this here so people don't commit there secret key by mistake ?

Reply:
I also wonder if config.py will hold more variables in the future and it'll be a mess. I think we're better off by not adding this change to the repo.